src/components/screens/Feed/CommunityAboutScreen.tsx

```diff
@@ -1,13 +1,34 @@
-import React from "react";
+import React, { useEffect } from "react";
 import { ScrollView, Text, useTheme, VStack } from "native-base";
 import FastImage from "@gkasdorf/react-native-fast-image";
 import { IconPlanet } from "tabler-icons-react-native";
 import { useTranslation } from "react-i18next";
 import RenderMarkdown from "../../common/Markdown/RenderMarkdown";
+import useCommunity from "../../../hooks/communities/useCommunity";
+import LoadingErrorView from "../../common/Loading/LoadingErrorView";
+import NotFoundView from "../../common/Loading/NotFoundView";
+import ModeratorList from "../../common/ModeratorList";
 
 function CommunityAboutScreen({ route }: { route: any }) {
   const { t } = useTranslation();
   const theme = useTheme();
+  const community = useCommunity(route.params.communityId);
+
+  useEffect(() => {
+    community.doLoad();
+  }, []);
+
+  if (community.communityNotFound) {
+    return <NotFoundView />;
+  }
+
+  if (community.communityLoading || !community.community) {
+    return null;
```

Comment:
We can use <LoadingView /> here instead of just returning null

Reply:
added Loading View